Documentation/api-reference.md

```diff
@@ -138,12 +138,16 @@ _tracker = [AEPMobileEdgeMedia createTracker];  // Use the instance for tracking
 
 ### createTrackerWithConfig
 
+> **Note**  
+> You can now adjust the tracking granularity for main and ad content. The allowed range of time interval in seconds for main content is `10 seconds <= main ping interval <=50 seconds` and for ad content is `1 second <= ad ping interval <=10 seconds`. The default interval is `10 seconds` for both main and ad content.
+
 Creates a media tracker instance based on the provided configuration to track the playback session.
 
 | Key | Description | Value | Required |
 | --- | --- | --- | --- |
-| config.channel | The channel name for media. Set this to overwrite the channel name configured in the Data Collection UI for media tracked with this tracker instance. | String | No |
-
+| "config.channel" | The channel name for media. Set this to overwrite the channel name configured in the Data Collection UI for media tracked with this tracker instance. | String | No |
+| "config.mainpinginterval" | Custom tracking interval for main content in seconds. Set this to override default ping interval for main content tracking. | Int | No |
```

Comment:
Overwrites the default main content tracking interval (in seconds). The value should be in the allowed range [10-50] seconds. The default value is 10 seconds.